test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProjectWithTransitiveDeps.cs

```diff
@@ -35,7 +35,7 @@ public void It_can_generate_all_runtimeconfig_files_to_output_folder()
             });
         }
 
-        [FullMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/3785")]
+        [FullMSBuildOnlyFact]
```

Comment:
Unskipping this test failed. Marking it with the successor issue. Details here: https://github.com/dotnet/sdk/issues/11008#issuecomment-2183528729